docs/cli.rst

```diff
@@ -1,11 +1,293 @@
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+..
+
+    Example: ``dev_org`` flow shows that it's composed of three subflows, ``dependencies``, ``deploy_unmanaged``, and ``config_dev``, and one task, ``snapshot_changes``. The tasks and flows making up the three subflows are also listed.
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+        when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Run Tasks and Flows
+*******************
+
+Execute a specific task or flow with the ``run`` command.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> [options]
+    $ cci flow run <name> --org <org> [options]
+
+This command runs the respective task or flow ``<name>`` against the org ``<org>``. (You can see a list of available orgs by running ``cci org list``.)
+
+    Example: The ``run_tests`` task executes Apex unit tests. Assuming there exists an org named ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``.
+
+
+
+Troubleshoot Errors
+-------------------
+
+Errors happen! That's why ``cci`` provides tools to extract error details so that they can be reported and triaged.
+
+
+Report Error Logs
+*****************
+
+The ``cci error gist`` command sends the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. For this feature to work you need to make sure that your `GitHub  service is set up with the proper scopes <https://cumu:lusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
+
+The gist includes the:
+
+* Current version of ``cci``
+* Current Python version
+* Path to the Python executable
+* ``sysname`` of the host (such as Darwin)
+* Machine name of the host (such as x86_64)
+* Most recent log file (cci.log) that CumulusCI has created.
+
+The URL for the gist is displayed in the terminal as output, and a web browser automatically opens a tab to the gist.
+
+
+The ``--debug`` Flag
```

Comment:
To confirm, everyone is cool with calling it the ``--help`` and ``--debug`` flag?

Reply:
@justinvholt huh? The info under this heading is only talking about `--debug`, not `--help`.

Reply:
@davisagli Earlier in the section, we discuss ``--help``. Previously ``--help`` and ``--debug`` were referred to as options. @Br4nd0R changed option to flag on his recent update. Just want to confirm that we're good with this change across the board.

Reply:
Oh, I see. A "flag" is a synonym for a specific kind of option that is boolean (either there or not) rather than taking a 2nd parameter (like `--org some_org_name`). I think either would be accurate; leaving it up to you which is clearer.

Reply:
Let's keep "flag" for now. I think this could be something we could revisit after the initial release if we want. I've noted it on the quip doc.